### DISCUSSION TITLE
There is a solution to the high-frequency squeaking of the speaker!

### DISCUSSION THREAD

**Discussion by Linkinterteid** [OnionUI/Onion#1314] [General]
The problem has already been solved by other people, I tried the build with this fix and it really works, no more high-frequency squeaking! Why can't this be implemented into the main system??? What's stopping you? Are there any conflicts with performance???

https://github.com/belmix/OnionRV/commit/a2f6da6b514ed3660be364d896dc4b7a1c6bddf0

**Comment by thomasklemenc** [comment]
This has been fixed by me in #1179 and will be included in 4.3.

The link you posted is just a (translated) fork of our code.
